Index: aegis/config/secret.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- aegis/config/secret.py	(revision 9058a994ad4492e6b2c214028cc83abff9f71ae5)
+++ aegis/config/secret.py	(date 1548408136000)
@@ -69,7 +69,7 @@
 
 # encrypt ( str or bytes ) return base64:str;
 def AES_ENCRYPT_BASE64(msg) -> str:
-    return re.sub(r'(\s|\n)+', '', base64.b64encode(AES_ENCRYPT(msg)).decode(encoding='utf-8'))
+    return re.sub(r'(\s|\n)+', '', base64.urlsafe_b64encode(AES_ENCRYPT(msg)).decode(encoding='utf-8'))
 
 
 # decrypt ( bytes ) return bytes;
@@ -82,14 +82,14 @@
 def AES_DECRYPT_BASE64(msg: str) -> str:
     msg = re.sub(r'(\s|\n)+', '', msg)
     msg += '=' * (-len(msg) % 4)
-    msg = base64.b64decode(bytes(msg, encoding='utf-8'))
+    msg = base64.urlsafe_b64decode(bytes(msg, encoding='utf-8'))
     return AES_DECRYPT(msg).decode(encoding='UTF-8')
 
 
 # decrypt ( base64:str ) return bytes;
 def AES_DECRYPT_BASE64Bytes(msg: str) -> bytes:
     msg += '=' * (-len(msg) % 4)
-    msg = base64.b64decode(bytes(msg, encoding='utf-8'))
+    msg = base64.urlsafe_b64decode(bytes(msg, encoding='utf-8'))
     return AES_DECRYPT(msg)
 
 from urllib.parse   import quote
